metadata.yaml

```diff
@@ -51,6 +51,9 @@ provides:
     interface: grafana_dashboard
   grafana-source:
     interface: grafana_datasource
+  remote-configurer:
+    interface: alertmanager_remote_configurer
```

Comment:
Please change from configurER to configurATION. 👍🏼

Reply:
Done.